spec/pom.xml

```diff
@@ -17,74 +17,88 @@
-        <status>DRAFT</status>
-        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
-        <revisiondate>${maven.build.timestamp}</revisiondate>
-    </properties>
+    <name>Jakarta XML Binding Specification</name>
 
     <scm>
         <connection>scm:git:git://github.com/eclipse-ee4j/jaxb-api.git</connection>
         <developerConnection>scm:git:git@github.com:eclipse-ee4j/jaxb-api.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jaxb-api.git</url>
         <tag>HEAD</tag>
     </scm>
-
     <distributionManagement>
         <site>
             <url>scm:git:git@github.com:eclipse-ee4j/jaxb-api.git</url>
         </site>
     </distributionManagement>
 
+    <properties>
+        <site.output.dir>${project.build.directory}/staging</site.output.dir>
+        <maven.site.skip>true</maven.site.skip>
+        <asciidoctorj.version>2.5.2</asciidoctorj.version>
+        <asciidoctorj.pdf.version>1.6.0</asciidoctorj.pdf.version>
+        <jruby.version>9.3.2.0</jruby.version>
+        <spec.name>jakarta-${project.artifactId}-${project.version}</spec.name>
+        <!-- status: DRAFT, BETA, etc., or blank for final -->
+        <status>DRAFT</status>
+        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
+        <revisiondate>${maven.build.timestamp}</revisiondate>
+    </properties>
+
     <build>
         <defaultGoal>package</defaultGoal>
         <plugins>
+            <!-- Sets minimal Maven version -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <version>3.0.0-M3</version>
+                <version>3.0.0</version>
                 <executions>
                     <execution>
-                        <id>enforce-versions</id>
+                        <id>enforce-maven</id>
                         <goals>
                             <goal>enforce</goal>
                         </goals>
                         <configuration>
                             <rules>
-                                <requireJavaVersion>
-                                    <version>(,1.9),[11,)</version>
-                                    <message>You need JDK8 or JDK 11 and newer</message>
-                                </requireJavaVersion>
+                                <requireMavenVersion>
+                                    <version>3.6.0</version>
+                                </requireMavenVersion>
                             </rules>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
+            
             <plugin>
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
-                <version>${asciidoctor.maven.plugin.version}</version>
+                <version>2.2.1</version>
```

Comment:
Why this one has to be different from others and defined here instead of by the property?

Reply:
Because it's a plugin here and not a dependency. The other plug-ins don't have their version defined by a property either, like e.g. the `maven-assembly-plugin`. This makes it more consistent (and similar to most other spec projects).

Reply:
Does `mvn versions:display-property-updates` see the new version available, if there is any?

Reply:
I always use `mvn versions` to periodically check projects for new versions, and it sees them indeed.